cdc/owner/ddl_manager.go

```diff
@@ -30,6 +29,10 @@ import (
 	"go.uber.org/zap"
 )
 
+// tableBarrierNumberLimit is used to limit the number
+// of tableBarrier in a single barrier.
+const tableBarrierNumberLimit = 256
+
 // globalDDLs is the DDLs that affect all tables in the changefeed.
```

Comment:
Can it be an allow list? E.g, we only allow DDL in the list to be executed in parallel. I think it's safer and prevents issues that TiDB add a new DDL can not be executed in parallel.

Reply:
done